rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -336,16 +336,16 @@ lv2_fs_object::lv2_fs_object(utils::serial& ar, bool)
 u64 lv2_file::op_read(const fs::file& file, vm::ptr<void> buf, u64 size)
 {
 	// Copy data from intermediate buffer (avoid passing vm pointer to a native API)
-	uchar local_buf[65536];
+	std::vector<uchar> local_buf(size);
```

Comment:
Simply allocate min(65536, size), this was a workaround for when passing some huge size such as above 4GB.

Reply:
Isn't this completely busted anyway? It seems to memcpy the whole size into the 65536 buffer...

Reply:
There was an std::min there.

Reply:
Fixed.